### ISSUE TITLE
Any way to use env variables in `config.toml`?

### ISSUE THREAD

**Issue by bzbug** [makeworld-the-better-one/amfora#308]
I have this line in my `config.toml`:

```toml
include = "/home/username/.config/amfora/cozy_dark.toml"
```

I'd like to be able to replace it with something like:

```toml
include = "$XDG_CONFIG_HOME/amfora/cozy_dark.toml"
```

or even this, for paths relative to the `amfora`'s config directory:

```toml
include = "cozy_dark.toml" # if the file is located in the same dir where the config.toml is located
```

**Comment by mntn-xyz** [comment]
I think tilde paths should work, so `~/.config/amfora/cozy_dark.toml` would be equivalent to your first example. That's not quite what you were asking for, but it may serve your needs. Is there any reason that a tilde path wouldn't work for your use case?

**Comment by bzbug** [comment]
Tried that again just now, `config.toml`:

```toml
include = "~/.config/amfora/cozy_dark.toml"
```

Result:

```
$ amfora
Config error: open ~/.config/amfora/cozy_dark.toml: no such file or directory
```

**Comment by mntn-xyz** [comment]
Ok, confirmed that this isn't done in all places that reference a path. It works with certificates, for example, see here: https://github.com/makeworld-the-better-one/amfora/blob/e3e1fc27cdd50657ff7b81278661733811e356e1/client/client.go#L114

What's missing is the call to homedir.Expand() where relative paths are needed.

**Comment by makeworld-the-better-one** [comment]
Could catch, it's definitely useful to support tildes in this location. Fixed in 3917402.
